requirements.txt

```diff
@@ -5,4 +5,5 @@ python-dateutil==2.8.0
 jinja2==2.10.1
 babel==2.7.0
 zimscraperlib>=1.0.1,<1.1
-kiwixstorage==0.1
+requests>=2.23.0
```

Comment:
I think the latest release is 2.23.0. How can we be sure to have <3.0? How about just `==2.23.0`?

Reply:
just use the string is gave you. It ensures it will be `>=2.23` (latest version now) and `<3.0` (next major release, hence API breaking). No `==2.23.0` won't work as other dependencies (storage for instance) might not use just that very same version.

Reply:
this has not been fixed